src/middleware/devtools.ts

```diff
@@ -27,21 +29,19 @@ type StoreSetStateWithAction<S> = S extends { getState: () => infer T }
 interface DevtoolsOptions {
```

Comment:
See if we can use the extracted `EnhancerOptions` type too (We'd also get the jsdoc for free). Something like...

```ts
interface DevtoolsOptions extends Pick<EnhancerOptions, "name" | "serialize"> {
  anonymousActionType?: string
}
```

Reply:
This works, but now TS users have to install some devDependencies (very counter-intuitive), which isn't very handy. Let's keep it as is.

Reply:
Yeah that's fair. One could write a postinstall script that fetches the `.d.ts` files from unpkg and writes it to the filesystem but I think that would be going too far.